tests/cpython-tests/exec.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+set -e
+make run-list TESTFILE=tests.passed
```

Comment:
I think we can fold these two lines into Makefile.

Reply:
runtest doesn't work well in case of multiple commands. We use this exec.sh trick in a few other tests as well.